src/components/TextField/TextField.tsx

```diff
@@ -522,11 +522,9 @@ export function TextField({
   }
 }
 
-function normalizeAutoComplete(autoComplete?: boolean | string) {
-  if (autoComplete === true) {
-    return 'on';
-  } else if (autoComplete === false) {
-    return 'off';
+function normalizeAutoComplete(autoComplete: string, name?: string) {
+  if (autoComplete === 'off' && !name) {
```

Comment:
I think the logic is if === off && there is a name --> Change to nope.
If no name and off, leave it alone.